test/e2e/framework/kubelet/stats.go

```diff
@@ -548,14 +548,14 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 	return result, utilerrors.NewAggregate(errs)
 }
 
-// GetMasterNodeLatest returns the latest resource usage of master and node.
-func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
+// GetControlPlaneNodeLatest returns the latest resource usage of control plane and node.
+func (r *ResourceMonitor) GetControlPlaneNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
 	result := make(ResourceUsagePerNode)
```

Comment:
here you rename GetMasterNodeLatest to GetControlPlaneNodeLatest. But I can not find where is calling GetControlPlaneNodeLatest

Reply:
Thank you for pointing this out. I will find the instance of GetMasterNodrLatest and refactoring it for control plane usage when I get back to my computer.